core/rawdb/databases_386.go

```diff
@@ -0,0 +1,16 @@
+package rawdb
```

Comment:
Do you need a build tag here?

Reply:
No, I think it's because the `_386` in the filename works like a build-tag, only `386`